liquibase-core/src/main/java/liquibase/Liquibase.java

```diff
@@ -316,6 +324,139 @@ public void update(Contexts contexts, LabelExpression labelExpression, boolean c
         });
     }
 
+    //
+    // Show summary information of the change sets which were skipped, and also
+    // a count of the change sets which were processed during this operation
+    //
+    private void showUpdateSummary(DatabaseChangeLog changeLog, StatusVisitor statusVisitor)
```

Comment:
Good this is private since we'll be moving all this code to UpdateCommandStep eventually and we don't want to have to support people overriding this function